ios/Magnet/Info.plist

```diff
@@ -35,8 +35,10 @@
 			</dict>
 		</dict>
 	</dict>
+	<key>NSLocationAlwaysUsageDescription</key>
+	<string>Project Magnet needs to check your location</string>
 	<key>NSLocationWhenInUseUsageDescription</key>
-	<string></string>
+	<string>Project Magnet needs to check your location</string>
```

Comment:
Now we need to ask for 'always usage description', I'm open to better wording you know I'm not the best there :P